src/ts/createTyperighterPlugin.ts

```diff
@@ -54,20 +59,21 @@ const createTyperighterPlugin = <TMatch extends IMatch>(
     expandRanges = expandRangesToParentBlockNode,
     matches = [],
     isActive = true,
+    ignoreMatch = () => false
```

Comment:
`() => false` appears in four places.  Can it be DRYed out and given a name?

Reply:
Great point, let's provide a default implementation.

Reply:
Did you?